flexmeasures/data/services/scheduling.py

```diff
@@ -204,6 +181,61 @@ def make_schedule(
     return True
 
 
+def load_custom_scheduler(scheduler_specs: dict) -> Tuple[Callable, str]:
+    """
+    Read in custom scheduling spec.
+    Attempt to load the Callable, also derive a data source name.
+
+    Example specs:
+
+    {
+        "module": "/path/to/module.py",  # or sthg importable, e.g. "package.module"
+        "function": "name_of_function",
+        "source": "source name"
+    }
+
+    """
+    assert isinstance(
+        scheduler_specs, dict
+    ), f"Scheduler specs is {type(scheduler_specs)}, should be a dict"
+    assert "module" in scheduler_specs, "scheduler specs have no 'module'."
+    assert "function" in scheduler_specs, "scheduler specs have no 'function'"
+
+    source_name = scheduler_specs.get(
+        "source", f"Custom scheduler - {scheduler_specs['function']}"
```

Comment:
The source name is being used, so we can later look up the schedule with `sensor.search_beliefs`.

Until now, we have simply assumed "Seita" as data source name and then looked that up ([see API endpoint](https://github.com/FlexMeasures/flexmeasures/pull/511/files#diff-26bb564e5c51a5dcb01d2939dfa624c47fd6d3e78f69451790767beca5a2dc76L521))

So I agree we probably should do better here, from now on.

Since we customized scheduling, we're saving the data source name on the job's meta attributes.

I believe you are proposing we save all three attributes on the job, correct?
And that we derive them from the custom scheduler module. Where is the author/organisation name coming from, though?

Another TODO: We also have to start saving this meta information on the jobs when using our internal schedulers.

I'll address this stuff in a separate commit once we have cleared up here what the organization name could be.